### PR TITLE
Update branding references

### PR DESCRIPTION
**Pull request by callmetwan**
Updates references from Kazoo to WorkTango

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ further defined and clarified by project maintainers.
 ### Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at [open-source@kazoohr.com]()]. All
+reported by contacting the project team at [open-source@WorkTangohr.com]()]. All
```
Comment (davidangel) [inline CONTRIBUTING.md]:
```suggestion
reported by contacting the project team at [open-source@worktango.com]()]. All
```

File: CLA.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+Thank you for your interest in contributing to open source software projects (“Projects”) made available by WorkTango, Inc. or its affiliates (“WorkTango”). This Contributor License Agreement (“Agreement”) sets out the terms governing any source code, object code, bug fixes, configuration changes, tools, specifications, documentation, data, materials, feedback, information or other works of authorship that you submit or have submitted, in any form and in any manner, to WorkTango in respect of any of the Projects (collectively “Contributions”). If you have any questions regarding this Agreement, please contact open-source@WorkTangohr.com.
```
Comment (davidangel) [inline CLA.md]:
```suggestion
Thank you for your interest in contributing to open source software projects (“Projects”) made available by WorkTango, Inc. or its affiliates (“WorkTango”). This Contributor License Agreement (“Agreement”) sets out the terms governing any source code, object code, bug fixes, configuration changes, tools, specifications, documentation, data, materials, feedback, information or other works of authorship that you submit or have submitted, in any form and in any manner, to WorkTango in respect of any of the Projects (collectively “Contributions”). If you have any questions regarding this Agreement, please contact open-source@worktango.com.
```

File: package.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+  "name": "@WorkTangohr/typeorm-pagination",
+  "repository": "github:WorkTangoHR/typeorm-pagination",
   "private": true,
   "version": "1.0.0-alpha",
   "engines": {
     "node": ">=14.0.0"
   },
   "main": "build/index.js",
-  "author": "Kazoo <engineering@kazoohr.com>",
+  "author": "WorkTango <engineering@WorkTangohr.com>",
```
Comment (davidangel) [inline package.json]:
```suggestion
  "author": "WorkTango <engineering@worktango.com>",
```

**Comment by callmetwan** [comment]
Thanks David! I noticed this too right as I pushed up :)

File: package.json [hunk 4]
```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@kazoohr/typeorm-pagination",
-  "repository": "github:KazooHR/typeorm-pagination",
+  "name": "@WorkTangohr/typeorm-pagination",
+  "repository": "github:WorkTangoHR/typeorm-pagination",
```
Comment (davidangel) [inline package.json]: Is this right? Do we have a separate GH account for WorkTango yet?

Comment (callmetwan) [reply package.json]: No, this is me just being hyper destructive with my find and replace

File: README.md [hunk 5]
```diff
@@ -82,7 +82,7 @@ For more information on how cursor-based pagination works, and how it's standard
 ### Examples
 
 ```ts
-import { findWithPagination } from "@kazoohr/typeorm-pagination";
+import { findWithPagination } from "@WorkTangohr/typeorm-pagination";
```
Comment (davidangel) [inline README.md]: Same question about whether this package name exists/works.

File: package.json (hunk 4 above)
Comment (callmetwan) [reply package.json]: Specifically I guess my regex filter to only markdown files when I did the find/replace didn't work 🙃

File: README.md (hunk 5 above)
Comment (peterbherman) [inline README.md]: Hah I know this isn't quite what it looks like in the code, but also maybe we should keep the lowercasing? 🤷 

```suggestion
import { findWithPagination } from "@worktango/typeorm-pagination";
```

**Review by davidangel — APPROVED** [review]
Hooray for hiring again!

This does make me wonder what sort of effort would be required to change our org name in github. Probably not a problem for today...

File: package.json [hunk 6]
```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@kazoohr/typeorm-pagination",
-  "repository": "github:KazooHR/typeorm-pagination",
```
Comment (peterbherman) [inline package.json]: Do we need to keep this one as KazooHR? That's our github org still.

File: package.json [hunk 7]
```diff
@@ -1,13 +1,13 @@
...
+  "name": "@WorkTangohr/typeorm-pagination",
```
Comment (peterbherman) [inline package.json]: Same nit [here](https://github.com/KazooHR/typeorm-pagination/pull/4/files#r1584858519) if you make that other change.

**Review by peterbherman — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (callmetwan) [reply README.md]: I switched it back to kazoohr for consistency

**Comment by callmetwan** [comment]
Silly mistakes fixed, only the needed bits are here now.

**Pull request merged**